extension.json

```diff
@@ -55,5 +55,17 @@
 		}
 	},
 
+	"RestRoutes": [
+		{
+			"path": "/wikibase-rdf/v0/map/{entity-id}",
+			"method": "POST",
+			"factory": "ProfessionalWiki\\WikibaseRDF\\Rest\\Handler\\Mappings::factory"
+		},
+		{
+			"path": "/wikibase-rdf/v0/map/{entity-id}",
+			"method": "GET",
+			"factory": "ProfessionalWiki\\WikibaseRDF\\Rest\\Handler\\Mappings::factory"
+		}
```

Comment:
I'm not sure we need an API to get the mappings for a single Entity. Did you add this to get them for the UI that displays the mappings on Entity pages? Ideally that UI gets them from something present in the page output, and part of the parser cache.

Reply:
I added the GET so that API users would be able to consume the information without having to scrape a page.  The spec seems to indicate that these mappings are per-entity (see "[Extension Concept](https://notepad.rhizome.org/wbsg-rdf-extension-workplan-r2#Extension-Concept)").

If you were to say that the mappings should be global, I would agree, but that introduces a UI issue -- does the UI only show relevant mappings for that page?

Reply:
We definitely need these APIs:

* write the mappings for an Entity - used by the UI
* list ALL mappings (for all Entities) - client asked for this

We _might_ also want an API to get the mappings for a specific Entity, like you started adding here, but that is not clear to me at the moment.

Reply:
done